Kudu.Core/Deployment/DeploymentStatusManager.cs

```diff
@@ -96,7 +96,7 @@ public DateTime LastModifiedTime
                     {
                         return DateTime.MinValue;
                     }
-                }, LockTimeout);
+                }, "Get Last Deployment Modified Time", LockTimeout);
```

Comment:
Since this is user visible, worth polishing a bit:

- don't capitalize every word
- use 'getting... ' instead of 'get... '. e.g. "Getting last deployment modified time"

Same general comment for other places